enigma/src/main/java/cuchaz/enigma/analysis/StructureTreeNode.java

```diff
@@ -90,6 +92,37 @@ public String toString() {
         return result;
     }
 
+    public String toHtml() {
+        List<String> modifiers = new ArrayList<>();
+
+        if (this.entry instanceof DefEntry<?>) {
+            AccessFlags access = ((DefEntry<?>) this.entry).getAccess();
+            boolean isInterfaceMethod = false;
+
+            if (this.entry instanceof MethodEntry && this.entry.getParent() instanceof ClassDefEntry) {
+                isInterfaceMethod = ((ClassDefEntry) this.entry.getParent()).getAccess().isInterface();
+            }
+
+            if (access.isStatic() && !access.isEnum()) {
+                // Static member, but not an enum constant
+                modifiers.add("static");
+            } else if (isInterfaceMethod && !access.isAbstract()) {
+                // Non-static default interface method
+                modifiers.add("default");
+            }
+
+            if (access.isAbstract() && !access.isInterface() && !isInterfaceMethod) {
+                // Abstract, but not an interface or an interface method (they're always abstract)
+                modifiers.add("abstract");
```

Comment:
Should exclude this from enum classes too.

Example:
```java
public enum SomeEnum {
  FIRST() {
    public String bark() {
      return "one";
    }
  },
  SECOND() {
    public String bark() {
      return "two";
    }
  };

  public abstract String bark();
}
```
Here the enum class is abstract but is implemented by each enum instance's anonymous class.

Reply:
Done in 591453e.